exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -158,7 +158,7 @@ func (ch *clickHouse) shardCountWatcher(ctx context.Context) {
 	ticker := time.NewTicker(ch.watcherInterval)
 	defer ticker.Stop()
 
-	q := `SELECT count() FROM system.clusters WHERE cluster='cluster'`
+	q := `SELECT count() FROM system.clusters`
```

Comment:
We are fine with re-initializing the in-memory map `timeSeries` for change in `system.clusters`. Should be fine because we do not expect it to happen often.